.pubnub.yml

```diff
@@ -2,6 +2,9 @@
 changelog: 
   - 
     changes: 
```

Comment:
Don't we need to update the `version: ` ?

Reply:
version was updated last time for the uuid changes, adding this change in the same version as 
those changes was not merged to master

Reply:
oh ok thanks